manifests/i/iTopDataRecovery/iTopDataRecovery/3.0.0.177/iTopDataRecovery.iTopDataRecovery.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: iTopDataRecovery.iTopDataRecovery
+PackageVersion: 3.0.0.177
+PackageLocale: en-US
+Publisher: iTop

```

Comment:
For some reason GitHub won't let me suggest changes, but the Publisher is `iTop Inc.`